database_manager/src/run_server.cpp

```diff
@@ -2575,17 +2569,39 @@ int main(int argc, char **argv) {
 
 
     initServer();
-
-    //ToasterFactReader factRdPdg(node, "pdg/factList");
```

Comment:
Why did you remove factList from pdg?
Also factList from agent monitor was already there...